sdk/identity/azure-identity/out/azure/identity/_credentials/client_credential.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for azure.identity._credentials.client_credential (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .._authn_client import AuthnClient
+from .._base import CertificateCredentialBase, ClientSecretCredentialBase
+from azure.core.credentials import AccessToken
+from typing import Any, Mapping
+
+class ClientSecretCredential(ClientSecretCredentialBase):
+    def __init__(self, client_id: str, secret: str, tenant_id: str, **kwargs: Mapping[str, Any]) -> None: ...
```

Comment:
Sometimes we do client_id, tenant_id, <otherstuff including secrets> and sometimes we put secret between client/tenant.  I prefer the former, but regardless consistency would seem desirable.

Reply:
We should do `tenant_id, client_id` whenever they are required. If they are not required, they should be `**kwargs` and not positional.

Reply:
Agreed. https://github.com/Azure/azure-sdk-for-python/issues/7951